person.rb

```diff
@@ -0,0 +1,39 @@
+require './nameable'
+require './capitalize_decorator'
+require './trimmer_decorator'
+
+class Person < Nameable
+  def initialize(name = 'unknown', age, parent_permission: true) # rubocop:todo Style/OptionalArguments
+    super()
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] A person can have many rentals. In order to implement this `has-many` relationship, please consider initializing `@rentals` array in your constructor, then create a `#add_rental` method which will add a rental 👍